### DISCUSSION TITLE
After uploading the csv file, I would like to get an idea of making a table.

### DISCUSSION THREAD

**Discussion by KoYoungWock** [refinedev/refine#5842] [Ideas]
I want to upload a csv file and make a table that can be edited (cell edit, row add) with its contents,
Can you give me an idea??

**Comment by BatuhanW** [comment]
Hey @KoYoungWock You can use `useEditableTable` in combination with csv import/export.

See our documentation:
- https://refine.dev/docs/ui-integrations/ant-design/hooks/use-editable-table
- https://refine.dev/docs/guides-concepts/import-export/
